force-app/main/default/classes/WeatherController.cls

```diff
@@ -0,0 +1,33 @@
+public with sharing class WeatherController {
+    @AuraEnabled(cacheable=true)
+    public static WeatherWrapper getWeather(String lat, String lon) {
+
+        try{
+
+            String apiKey = 'a7b5f1c8f58b997bf986d8d2ebb64d81';
```

Comment:
this is a classical example of what we use as constants

prefere something like this:

```
static final Integer PRIVATE_INT_CONST = 200;
   static final Integer PRIVATE_INT_CONST2; 
```